bpf/bpf_lxc.c

```diff
@@ -1941,6 +1941,119 @@ int tail_ipv4_policy(struct __ctx_buff *ctx)
 	return ret;
 }
 
+static __always_inline bool
+ipv4_to_endpoint_is_hairpin_flow(struct __ctx_buff *ctx, struct iphdr *ip4, __u32 __maybe_unused src_sec_identity)
+{
+	struct ipv4_ct_tuple tuple = {};
+	struct lb4_backend *backend __maybe_unused;
```

Comment:
```suggestion
	struct lb4_backend *backend;
```

I think I added the attribute in my version because I commented out code in the middle to focus on the second lookup. But it's not unused